### PR TITLE
restore doc updated

### LINKED ISSUE #1685: Restore docs should state you must use the same version that was backed up
@bgrissin commented

Add to the documentation for DDC (UCP and DTR backup and restore) that performing restore require restoration to the same release that the backups were acquired from

All DDC releases for UCP and DTR

### PR DESCRIPTION
**Pull request by prashant171992**
### Proposed changes

Updated the DDC documentation to mention use of same version that was backed up during restore (UCP & DTR).

### Related issues (optional)

Fixes #1685

### CODE REVIEW

File: datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ $ tar -tf /tmp/backup.tar
 
 ## Restore DTR data
 
+Performing restore require restoration to the same release that the backups were
```
Comment (mstanleyjones) [inline datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md]: Can you please update the phrasing a bit? Something like "DTR data must be restored to the same release version where the backups were obtained." Or something like that. What do you think?

Comment (prashant171992) [inline datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md]: @mstanleyjones You are right , I made the changes accordingly.

File: datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md [hunk 2]
```diff
@@ -86,6 +86,7 @@ $ tar -tf /tmp/backup.tar
 
 ## Restore DTR data
 
+DTR data must be restored to the same release version where the backups were obtained.
```
Comment (joaofnfernandes) [inline datacenter/dtr/2.0/high-availability/backups-and-disaster-recovery.md]: The sentence is a bit confusing 😢 Can you rephrase to something like:
"When restoring, make sure you use the same version of the `docker/dtr` image that you've used to create the backup".

File: datacenter/ucp/1.1/high-availability/backups-and-disaster-recovery.md [hunk 3]
```diff
@@ -61,6 +61,8 @@ Enter passphrase: secret
 
 ## Restore command
 
+DTR data must be restored to the same release version where the backups were obtained.
```
Comment (joaofnfernandes) [inline datacenter/ucp/1.1/high-availability/backups-and-disaster-recovery.md]: After rephrasing this, can you move this sentence to line 68? That way we will start by explaining what the `restore` command does, and then tell that you should run the same version.

**Comment by prashant171992** [comment]
@joaofnfernandes done with the recommended changes, please check cc : @mstanleyjones
